src/SchemaDesign/SchemaList.tsx

```diff
@@ -0,0 +1,62 @@
+import Typography from '@mui/material/Typography'
+import Button from '@mui/material/Button'
+import Paper from '@mui/material/Paper'
+import Box from '@mui/material/Box'
+import Stack from '@mui/material/Stack'
+import Grid from '@mui/material/Grid'
+import Link from '@mui/material/Link'
+import { useGetSchemas } from '../../data/schema'
+
+export default function SchemaList() {
+  const { schemas: uploadSchemas, isSchemasLoading: isUploadSchemasLoading } = useGetSchemas('UPLOAD')
+  const { schemas: deploymentSchemas, isSchemasLoading: isDeploymentSchemasLoading } = useGetSchemas('DEPLOYMENT')
+
+  return (
+    <Box>
+      {!isUploadSchemasLoading && !isDeploymentSchemasLoading && (
+        <Grid container spacing={2}>
+          <Grid item sm={12} md={6} sx={{ width: '100%' }}>
+            <Paper sx={{ mt: 2, mb: 2, p: 2 }}>
```

Comment:
Fine as is, but could be shortened to `sx={{ my: 2, p: 2 }}`

Reply:
Didn't even know `my` existed! Very useful to know